### DISCUSSION TITLE
How To Improve Nonhuman Voice Tone Into Human-like Voice?

### DISCUSSION THREAD

**Discussion by jimmyxp01** [coqui-ai/TTS#1795] [Ideas]
I have a generated voice. It doesn't sound like a human. 

So is there any option to improve that voice like a human one?

Is there any model about is this issue? Is or Any solution?

**Comment by p0p4k** [comment]
Use larger noise free dataset.

**Reply by jimmyxp01** [reply]
Any example? Which one is best?

**Reply by p0p4k** [reply]
https://github.com/coqui-ai/open-speech-corpora
